### ISSUE TITLE
Mal'Ganis the Friendly. Another broken script with no attackable boss

### ISSUE THREAD

**Issue by taichouVu** [dalaranwow/dalaran-wow#5957]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Mal'ganis can't be attacked.

**Expected behaviour**: Tell us what should happen instead
Mal'Ganis should agro when Arthus gets done talking

**Steps to reproduce the problem**:

1. Baby sit Arthus as he runs past packs of Mobs in the Gauntlet. Another bug btw.
2. Wait for him to engage Mal'ganis. and keep waiting
3. Die of old age waiting. Or Dance in fromt of Mal'Ganis 
![WoWScrnShot_052621_144038](https://user-images.githubusercontent.com/79895201/119735354-09adeb80-be31-11eb-8771-9da610911ec2.jpg)

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by taichouVu** [comment]
05/27/2021 -Succeeded in complete the Heroic. I am guessing the bug only happens if you die during the encounter with Mal'Ganis. Still had to clear the gauntlet then come back for Arthus so he doesn't run past the mobs.

**Comment by Lothloryen** [comment]
Hello there,

Thank you for the report, this is the first time I've seen this - I will take a look.
